configs/container.config

```diff
@@ -22,6 +22,7 @@ process {
     withLabel: virsorter        { container = 'multifractal/virsorter:0.1.2' }
     withLabel: phigaro          { container = 'multifractal/phigaro:0.5.2' }
     withLabel: pharokka         { container = 'multifractal/pharokka:v1.7_seqkit' }
+    withLabel: phage_plot       { container = 'nanozoo/pycirclize:3.12--48de646' }
```

Comment:
i assume node and local config is missing?

Reply:
ah i need to remove this.. 
pharokka container is doing this but I need to add them to local and node